api-gateway/src/main/java/com/petclinic/bffapigateway/utils/Utility.java

```diff
@@ -0,0 +1,10 @@
+package com.petclinic.bffapigateway.utils;
+
+
+import org.springframework.http.server.reactive.ServerHttpRequest;
+
+public class Utility {
+    public static String getSiteURL(ServerHttpRequest request) {
+        return request.getURI().toString();
```

Comment:
can you send me a message explaining what this does?

Reply:
With the ServerHttpRequest you can get the the URL, but in this case I don't need it haha. This was used before to send the link in the email, but I had to remove it.